.changeset/lemon-oranges-promise.md

```diff
@@ -0,0 +1,7 @@
+---
+"@keystone-6/fields-document": major
+"@keystone-6/cloudinary": major
+"@keystone-6/auth": major
+---
+
+Use `g` export instead of `graphql` from `@keystone-6/core`. The only breakage is that you must be on the version of `@keystone-6/core` that includes the `g` export or newer
```

Comment:
Note that this changeset has major changes but the `@keystone-6/core` changeset is only a minor since it's not breaking to add the new name and keep the old one around but _using_ the new name is breaking since people _must_ update `@keystone-6/core` otherwise these packages will not work. While this would be easily avoidable by using the old version, I don't think that complexity is worth it since people will already be forced to use all the new versions together because of the changes for `@keystar/ui`.